android-stellar-sdk/src/main/java/org/stellar/sdk/responses/TransactionResponse.java

```diff
@@ -107,6 +120,33 @@ public Memo getMemo() {
     return memo;
   }
 
+  public List<Operation> getOperations() {
+    String envelopeXdr = getEnvelopeXdr();
+    try {
+      Transaction transaction = extractTransaction(envelopeXdr);
+      org.stellar.sdk.xdr.Operation[] xdrOperations = transaction.getOperations();
+      ArrayList<Operation> operationsList = new ArrayList<>(xdrOperations.length);
+      for (org.stellar.sdk.xdr.Operation xdrOperation : xdrOperations) {
+        operationsList.add(Operation.fromXdr(xdrOperation));
+      }
+      return operationsList;
+    } catch (UnsupportedEncodingException e) {
+      e.printStackTrace();
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    return null;
+  }
+
+  private Transaction extractTransaction(String envelopeXdr) throws IOException {
```

Comment:
now you can add unit test for extractTransaction:)

Reply:
unit test already add at `TransactionDeserializerTest` :grinning: